Components/alert.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Rhythm UI |Docs |Alert</title>
+    <link rel="shortcut icon" href="/favicon.ico?v=2" type="image/x-icon" />
+
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+    />
+    <link rel="stylesheet" href="/index.css" />
+    <link rel="stylesheet" href="/docs.css" />
+    <link rel="stylesheet" href="/Components-CSS/alert.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav class="header">
+        <a class="logo" href="/index.html">
+          <img width="80" src="/images/music.svg" alt="logo" />
+          <span><strong> RHYTHM UI</strong></span>
+        </a>
+        <div class="search-container">
+          <div class="search-box search">
+            <i class="fas fa-search"></i>
+
+            <span>
```

Comment:
Input and span both are inline elements, Also you don't have any classes on span here, so it's better to avoid empty wrappers with no meaning.